electron-app/src/server/websites/megalodon/megalodon.service.ts

```diff
@@ -33,6 +34,7 @@ import { FileManagerService } from 'src/server/file-manager/file-manager.service
 import * as fs from 'fs';
 import { tmpdir } from 'os';
 import * as path from 'path';
+import SubmissionPartEntity from 'src/server/submission/submission-part/models/submission-part.entity';
```

Comment:
nit: for future prs please remember to use relative paths for imports

Reply:
👍 Will prob want to do a pass over the codebase and correct a few of them at some point.